graylog2-server/src/main/java/org/graylog2/audit/AuditEventSender.java

```diff
@@ -14,16 +14,16 @@
  * You should have received a copy of the GNU General Public License
  * along with Graylog.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.graylog2.auditlog;
+package org.graylog2.audit;
 
 import java.util.Map;
 
-public interface AuditLogger {
-    void success(String subject, String action, String object);
+public interface AuditEventSender {
+    void success(AuditActor actor, AuditEventType type);
 
-    void success(String subject, String action, String object, Map<String, Object> context);
+    void success(AuditActor actor, AuditEventType type, Map<String, Object> context);
```

Comment:
I'd add a couple of convenience methods here, to make the call sites a bit more concise:

With
```Java
default void success(String actor, String type, Map<String, Object> context) {
    success(AuditActor.user(actor), AuditEventType.create(type), context);
}
```
it would be much easier to read on the call sites.

There are a couple of more options, too, such as allowing to append a varargs of Object pairs to implicitly create the map (or just a single String, Object pair)